docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# How to contribute
+
+Thanks for helping out!
+
+## Committing
+
+We try to follow the "[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)" convention, combined with the "[Angular Commit Message format](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#-commit-message-format)".
+
+When committing your changes, provide a commit message that starts with an appropriate keyword:
+
+- `feat`: new feature for the user
+- `fix`: bug fix for the user
+- `docs`: changes to the documentation
+- `style`: formatting, missing semi colons, etc; no production code change
+- `refactor`: refactoring production code, eg. renaming a variable
+- `test`: adding missing tests, refactoring tests; no production code change
+- `chore`: cleanups, version updates etc; no production code change
+
+Add an Azure DevOps task ID at the end of the commit message.  
+For example: "`feat: new feature added to the profile page AB#123456`".
+
+### Updating dependencies
+
+Most (development-)dependencies in this repository are monitored by the GitHub [Dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates) service, to keep them up-to-date.
+The configuration of these updates is in [`.github/dependabot.yml`](../.github/dependabot.yml).  
+Unfortunately most individual dependencies are 'linked' to related dependencies that need to stay 'in sync'.
+
+> [!NOTE]  
+> `Sheetjs` is not monitored by Dependabot. Check the latest version of `Sheetjs`: [![`Sheetjs` latest version](https://img.shields.io/badge/dynamic/xml?url=https%3A%2f%2fgit.sheetjs.com%2fsheetjs%2fsheetjs%2ftags.rss&query=.%2f%2fchannel%2fitem%5B1%5D%2ftitle&logo=microsoftexcel&logoColor=white&label=sheetjs&color=lightgreen)](https://git.sheetjs.com/sheetjs/sheetjs/tags)
+
+Interface dependencies:
+
+To update all Angular and ESLint related dependencies together, run (in each individual interface's directory):
+
+    npm run upgrade:angular
+
+All related changes will be handled by the Angular CLI, but need to be checked afterwards with `lint`, `test` commands and local testing.
```

Comment:
These sections feel a bit out-of-place here now, but I don't know where in the main README they would fit 'better' either... ;)